supported_devices_platforms_md.sh

```diff
@@ -90,17 +90,17 @@ echo "| 18   | Accton    	| AS9716-32D     		| Broadcom    | Tomahawk 3        |
-echo "| 23   | Arista        | 7050QX-32S     		| Broadcom    | Trident 2         | 32x40G                  | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 24   | Arista        | 7050CX3-32S     		| Broadcom    | Trident 3         | 32x100G + 2x10G         | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 25   | Arista        | 7060CX-32S     		| Broadcom    | Tomahawk          | 32x100G + 2x10G         | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 26   | Arista        | 7060DX4-32       		| Broadcom    | Tomahawk 3        | 32x400G + 2x10G         | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 27   | Arista        | 7060PX4-32       		| Broadcom    | Tomahawk 3        | 32x400G + 2x10G         | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 28   | Arista        | 7170-32CD       		| Barefoot    | Tofino            | 32x100G + 2x10G         | [SONiC-Aboot-Barefoot]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 29   | Arista        | 7170-64C       		| Barefoot    | Tofino            | 64x100G + 2x10G         | [SONiC-Aboot-Barefoot]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 30   | Arista        | 7260CX3-64     		| Broadcom    | Tomahawk 2        | 64x100G + 2x10G         | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 31   | Arista        | 7280CR3-32D4   		| Broadcom    | Jericho 2         | 32x100G + 4x400G        | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
-echo "| 32   | Arista        | 7280CR3-32P4   		| Broadcom    | Jericho 2         | 32x100G + 4x400G        | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-barefoot.swi/')) |" >> supported_devices_platforms.md
+echo "| 22   | Arista        | 7050QX-32      		| Broadcom    | Trident 2         | 32x40G                  | [SONiC-Aboot-Broadcom]($(echo "${ARTF_BFT}" | sed 's/format=zip/format=file\&subpath=\/target\/sonic-aboot-broadcom.swi/')) |" >> supported_devices_platforms.md
```

Comment:
actually ARTF_BFT points to Barefoot, while ARTF_BRCM is what points to broadcom

Reply:
can u please regenerate the PR with the fix